src/containers/DataTable/Table.js

```diff
@@ -47,7 +54,7 @@ const TableCell = ({ obj, header, idx, showJson, jsonTheme }) => {
 
 	else if (header.link) {
 		const pathname = header.link.pathname + obj[header.link.key];
-		cell = obj[header.key] ? <Link to={{ pathname }} className="data-table-link">{icon} {obj[header.key]}</Link> : "-";
+		cell = obj[header.key] ? <Link style={ textLinkStyle } to={{ pathname }} className="data-table-link">{icon} {obj[header.key]}</Link> : "-";
```

Comment:
May I also ask you to break lines between props. We should break lines when there are 3 or more props for one component. If we write oneliners we may make code unreadable. BTW I always forget about this myself.
Otherwise it's good and approved.